UPGRADING.md

```diff
@@ -0,0 +1,24 @@
+- [Upgrading OpenSearch Python Client](#upgrading-opensearch-python-client)
+  - [Upgrading to >= 2.2.0](#upgrading-to->=-2.2.0)
+    - [Features from High-Level Python Client](#features-from-high-level-python-client)
+    - [API Implementation differences for existing opensearch-dsl-py users](#api-implementation-differences-for-existing-opensearch-dsl-py-users)
+
+
+# Upgrading OpenSearch Python Client
+
+## Upgrading to >= 2.2.0
+
+opensearch-py now includes [opensearch-dsl-py](https://pypi.org/project/opensearch-dsl/) features. opensearch-dsl-py was merged into opensearch-py preserving backwards compatibility with the previous opensearch-py version. (Refer [link](https://github.com/opensearch-project/opensearch-py/pull/287))
+
+
+### Features from High-Level Python Client
+
+opensearch-py functionalities that already exist are not altered. Only the following areas have new feature additions.
```

Comment:
Can you make this some sort of a bulleted list? Or something that shows "following areas" as areas?

Reply:
Removed following areas line. Thanks for correcting.